core/templates/dev/head/css/oppia.css

```diff
@@ -3901,6 +4200,46 @@ md-card.preview-conversation-skin-supplemental-card {
   right: 0;
 }
 
+/* Styles for Settings tab */
+.oppia-settings-container .has-error .form-text {
+  color: #a94442;
+}
+
+.oppia-settings-container .form-text {
+  color: #737373;
+  display: inline-block;
+  margin-top: 5px;
+  margin-bottom: 15px;
+}
+
+.oppia-settings-container .col-sm-10 {
+  max-width: 100%;
+  padding-right: 0;
+}
+.oppia-settings-container .col-sm-10  > select {
+  width: 85%;
+}
+.oppia-settings-container .form-group > label {
+  min-height: 35px;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.oppia-advanced-features-card .oppia-on-off-switch-label {
```

Comment:
Why are these still here? Please remove unused rules (as well as references to classes which have no CSS rule specified).

Reply:
Removed it also, looking for unused CSS Thanks!